scene/gui/option_button.cpp

```diff
@@ -30,10 +30,44 @@
 
 #include "option_button.h"
 
+#include "core/config/project_settings.h"
 #include "scene/theme/theme_db.h"
 
 static const int NONE_SELECTED = -1;
 
+void OptionButton::gui_input(const Ref<InputEvent> &p_event) {
+	Ref<InputEventKey> k = p_event;
```

Comment:
Why willingly increase the refcount of `p_event` by making an explicit copy?

Reply:
It's casting it to `InputEventKey` to filter only key events, `p_event` can be any kind of event.